client/state/jetpack/onboarding/schema.js

```diff
@@ -0,0 +1,12 @@
+/** @format */
+
+export const jetpackOnboardingCredentialsSchema = {
```

Comment:
I think we can remove the named export now. And import it with a shorter name perhaps 😉